next.clayui.com/content/docs/components/select.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: "Select"
+description: "A form control element used to select from several provided options and enter data."
+packageNpm: "@clayui/select"
+packageVersion: "3.0.0"
+packageStatus: "Beta"
+---
+
+import {Select, SelectWithOption} from '../../../src/components/clay/Select';
+
+<div class="nav-toc">
+
+- [Variants and Best Pratices](#variants-and-best-pratices)
+    - [SelectWithOption](#selectwithoption)
+- [API](#api)
+
+</div>
+
+<div class="clay-site-alert alert alert-info">
+	Check the <a href="https://liferay.design/lexicon/">Lexicon</a> <a href="https://liferay.design/lexicon/core-components/forms/selector/">Selectors Pattern</a> for a more in-depth look at the motivations and proper usage of this component.
```

Comment:
Any idea if its possible to use markdown inside of html blocks? Im assuming not, but wasn't sure if we could just use a markdown link instead of an anchor tag.

Reply:
Considering our current implementation on Gatsby isn't possible
<img width="854" alt="Screen Shot 2019-06-17 at 19 20 32" src="https://user-images.githubusercontent.com/7663513/59640613-ff5b2100-9134-11e9-821d-65fc5599fd0d.png">
<img width="839" alt="Screen Shot 2019-06-17 at 19 20 11" src="https://user-images.githubusercontent.com/7663513/59640614-fff3b780-9134-11e9-9a5b-276c4dec9ee1.png">